src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/CourtService.kt

```diff
@@ -246,14 +252,20 @@ class CourtService(
   }
 
   private fun toVideoLinkAppointmentDto(scheduledAppointment: ScheduledAppointmentDto?) =
-    scheduledAppointment?.let {
+    scheduledAppointment?.takeIf { hasAnEndDate(it) }?.let {
       VideoLinkBookingResponse.LocationTimeslot(
         locationId = it.locationId,
         startTime = it.startTime,
-        endTime = it.endTime
+        endTime = it.endTime!!
       )
     }
 
+  private fun hasAnEndDate(it: ScheduledAppointmentDto): Boolean {
```

Comment:
Could be an extension function. Don't need it though.

Reply:
ooh that would be nice